### PR TITLE
[rabbitmq-c] initial integration

### PR DESCRIPTION
**Pull request by 0x34d**
`rabbitmq-c`  aka `librabbitmq`.

@alanxz thoughts.

Signed-off-by: Arjun Singh <ajsinghyadav00@gmail.com>

### CODE REVIEW

File: projects/rabbitmq-c/project.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+homepage: "https://github.com/alanxz/rabbitmq-c"
+language: c
+primary_contact: "alan.antonuk@gmail.com"
+auto_ccs:
+  - "ajsinghyadav00@gmail.com"
+fuzzing_engines:
+  - libfuzzer
+  - afl
+  - honggfuzz
```
Comment (jonathanmetzman) [inline projects/rabbitmq-c/project.yaml:9]: No need to specify these, they are all defaults.

**Review by jonathanmetzman — APPROVED** [review]
LGTM.
I'm gonna ask the committee if we'll accept this?

**Comment by jonathanmetzman** [comment]
> `rabbitmq-c` aka `librabbitmq`.
> 
> @alanxz thoughts.
> 
> Signed-off-by: Arjun Singh [ajsinghyadav00@gmail.com](mailto:ajsinghyadav00@gmail.com)

This is the client right? It's not the main tool.
I'm gonna make this a draft until upstream approves.

**Comment by alanxz** [comment]
Would be happy to have rabbitmq-c have regular fuzzing.

This LGTM.

**Pull request merged**